lib/wab.rb

```diff
@@ -4,6 +4,7 @@ module WAB
 end
 
 require 'wab/controller'
+require 'wab/configuration'
```

Comment:
This should not be part of the WAB module but in WAB::Impl. It is not required anywhere else. With the exception of the default Controller behavior all the other WAB module classes are duck-types or used to define the API like errors.

Reply:
That makes sense.. the config is ultimately for `WAB::Impl::Shell`..